protocols/ledger/ledger_test.go

```diff
@@ -0,0 +1,142 @@
+package ledger
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/google/go-cmp/cmp"
+	"github.com/statechannels/go-nitro/channel/state"
+	"github.com/statechannels/go-nitro/channel/state/outcome"
+	"github.com/statechannels/go-nitro/protocols"
+	"github.com/statechannels/go-nitro/types"
+)
+
+type actor struct {
+	address     types.Address
+	destination types.Destination
+	privateKey  []byte
+}
+
+var alice = actor{
+	address:     common.HexToAddress(`0xF5A1BB5607C9D079E46d1B3Dc33f257d937b43BD`),
+	destination: types.AddressToDestination(common.HexToAddress(`0xF5A1BB5607C9D079E46d1B3Dc33f257d937b43BD`)),
+	privateKey:  common.Hex2Bytes(`caab404f975b4620747174a75f08d98b4e5a7053b691b41bcfc0d839d48b7634`),
+}
+
+var bob = actor{
+	address:     common.HexToAddress(`0x760bf27cd45036a6C486802D30B5D90CfFBE31FE`),
+	destination: types.AddressToDestination(common.HexToAddress(`0x760bf27cd45036a6C486802D30B5D90CfFBE31FE`)),
+	privateKey:  common.Hex2Bytes(`62ecd49c4ccb41a70ad46532aed63cf815de15864bc415c87d507afd6a5e8da2`),
+}
+
+func TestCreateLedger(t *testing.T) {
+	ledgerManager := NewLedgerManager()
+	left := outcome.Allocation{Destination: alice.destination, Amount: big.NewInt(3)}
+	right := outcome.Allocation{Destination: bob.destination, Amount: big.NewInt(2)}
+
+	ledger, err := ledgerManager.CreateTestLedger(left, right, &alice.privateKey, 0)
+	if err != nil {
+		t.Error(err)
+	}
+
+	expectedState := state.State{
+		ChainId:           big.NewInt(9001),
+		Participants:      []types.Address{alice.address, bob.address},
+		ChannelNonce:      big.NewInt(0),
+		AppDefinition:     types.Address{},
+		ChallengeDuration: big.NewInt(45),
+		AppData:           []byte{},
+		Outcome: outcome.Exit{outcome.SingleAssetExit{
+			Allocations: outcome.Allocations{left, right},
+		}},
+		TurnNum: 0,
+		IsFinal: false,
+	}
+
+	gotState := ledger.SignedStateForTurnNum[0].State()
+	if diff := cmp.Diff(expectedState, gotState); diff != "" {
+		t.Errorf("TestCreateLedger: ledger state mismatch (-want +got):\n%s", diff)
+	}
```

Comment:
```suggestion
	if ledger.ChannelNonce.Cmp(big.NewInt(0)) != 0 {
		t.Error("TestCreateLedger: initial ledger channel should use the 0 nonce")
	}
```

the `state` check that's erased wasn't very dry - just checking that the init data is the same in both files, but not clear if that's useful.

The added check compliments the below check on incrementing nonces

Reply:
Since this is just a `test` function I agree!